test/system/files_frontend_test.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative "../test_helper"
+
+class FilesFrontendTest < ApplicationSystemTestCase
+
+  setup do
+    @site = comfy_cms_sites(:default)
+  end
+
+  def test_site_file_drag_and_drop
```

Comment:
seems like we can't test this due to chromedriver limitation. leaving skip but removing code.